README.md

```diff
@@ -16,22 +16,29 @@ The assumption is that if you are applying to software companies in Japan you wi
 | --- | --- | --- | --- |
 | [Amazon Japan](https://www.amazon.jobs/en/locations/tokyo-area-japan?base_query=&job_count=10&result_limit=10&sort=relevant&location%5B%5D=tokyo-area-japan&cache) | Shinagawa-ku, Tokyo | Multiple sectors | |
 | [Bizreach](http://www.bizreach.co.jp/recruit/) | Shibuya-ku, Tokyo | B2B Services, Technology R&D | |
+| [Cookpad](https://recruit.cookpad.com/) | Ebisu, Tokyo | Web, Mobile | |
+| [CyberAgent](https://www.cyberagent.co.jp/recruit/) |  Tokyo | Ads, B2B Services | |
 | [DeNa](http://dena.com/intl/careers/positions/) | Shibuya-ku, Tokyo | Primarily Mobile | |
 | [DMM](http://www.dmm.com/recruit/) | Tokyo | E-Commerce (Video), Mobile, Education | |
 | [Dwango](http://dwango.co.jp/recruit/) | Tokyo | Telecommunications & Media | |
+| [Fast Retailing](https://www.fastretailing.com/employment/ja/) | Tokyo | Retail | |
 | [GMO](http://recruit.gmo.jp/) | Tokyo & Other? | Multiple Sectors | |
 | [Google](https://www.google.com/about/careers/locations/tokyo/) | Minato-ku, Tokyo | Multiple Sectors | No |
+| [GREE](http://jobs.gree.net/jp/ja/career/) | Minato-ku, Tokyo | Social | No |
 | [GungHu](http://www.gungho.co.jp/recruit/) | Tokyo | Mobile | |
 | [IBM](http://www-07.ibm.com/employment/jp/cp01/) | Tokyo | Software, IT, Consulting | |
 | [i-mobile](https://www.i-mobile.co.jp/recruit/) | Shibuya-ku, Tokyo | Advertising | |
 | [Interspace](https://www.interspace.ne.jp/recruit/offer/) | Minato-ku, Tokyo | Advertising | |
+| [LINE](https://linecorp.com/ja/career/ja/all) | Fukuoka & Shibuya-ku, Tokyo | Mobile | |
 | [M3](https://corporate.m3.com/recruit/job/) | Minato-ku, Tokyo | Healthcare | |
 | [Microsoft](http://microsoft-college.jp/) | Shibuya-ku, Tokyo | Multiple Sectors | No |
-| [Naver](https://linecorp.com/ja/career/ja/all) | Shibuya-ku, Tokyo | Mobile | |
```

Comment:
I think there may be. But they spun off LINE into it's own corp so I'd say it's fair to list it as `LINE` on this line.